scripts/install-chef-suse.sh

```diff
@@ -287,11 +304,17 @@ Aborting: Administration Server is already deployed.
 If you want to run the installation script again,
 then please remove the following file:
 
-    /opt/dell/crowbar_framework/.crowbar-installed-ok
+    $crowbar_install_dir/crowbar-installed-ok
 EOF
     exit 1
 fi
 
+if [ -f $crowbar_install_dir/crowbar-install-failed ] || [ "$CROWBAR_WIZARD_MODE" ]; then
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
+    sqlite3 /opt/dell/crowbar_framework/db/production.sqlite3 "delete from proposals; delete from proposal_queues; vacuum;"
```

Comment:
If we need to keep the `sessions` table we can migrate down to the `sessions` migration and migrate up again. That way we can be sure that we got a clean database.